panel/src/components/Misc/Icon.vue

```diff
@@ -6,15 +6,18 @@
     :data-back="back"
     :data-size="size"
     :class="'k-icon k-icon-' + type"
+    :style="{ background: toColor(back) }"
```

Comment:
Wouldn't it make sense to just use `$helper.color(back)` here?

Reply:
You mean not putting the helper in a `css` namespace/object and actually registering it as a global helper?

Reply:
ok yes

Reply:
Do you think we need to put it in a css namespace at all? Anyway, I really think it would be good to have it as a global helper. This is super useful for plugin devs as well.